bump_version

```diff
@@ -24,7 +24,11 @@ fi
 shared_version_file="./SharedVersion.cs"
 build_file="./build.yaml"
 
-web_branch="$( git branch 2>/dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/\1/' )"
+if [[ -z $2 ]]; then
+    web_branch="$( git branch 2>/dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/\1/' )"
```

Comment:
git 2.22 added `--show-current ` to the branch command to achieve the same. Though I'm not sure it belongs here as 2.22 is releatively recent.

Reply:
It's about time lol! But yea that's still very recent, I think Buster is at 2.18 or so, so I'd want to leave it for a while yet.